app/src/main/java/com/github/palFinderTeam/palfinder/meetups/MeetUp.kt

```diff
@@ -156,6 +159,7 @@ data class MeetUp(
         const val HAS_CAPACITY = "hasMaxCapacity"
         const val CAPACITY = "capacity"
         const val ICON = "icon"
+        const val MARKER = "marker"
```

Comment:
What is exactly this ID? Is it the ID of an image in the library of these marker icons, or is it the ID number of the picture inside the android folders?

Reply:
It's the id of the image in a local icon pack